Source/Meadow.Foundation.Core/Motors/BidirectionalDcMotor.cs

```diff
@@ -46,6 +74,7 @@ public void Stop()
     public void Clockwise()
```

Comment:
Could we call these `StartClockWise()` and `StartCounterClockWise()`? it'll improve discoverability using intellisense when users type `start...` like we do on Led animations (i.e. StartBlink, StartPulse, etc.)